sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample01_HelloWorld.md

```diff
@@ -32,6 +32,37 @@ string body = receivedMessage.Body.ToString();
 Console.WriteLine(body);
 ```
 
+### Send and receive a message using topics and subscriptions
+
+As discussed in the [Key concepts section](https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/servicebus/Azure.Messaging.ServiceBus#key-concepts) of the README, Service Bus supports topics and subscriptions that are useful in pub/sub scenarios. When using topics and subscriptions, each subscription will get its own copy of each message that is delivered to the topic. So completing a message in one subscription won't impact what happens to the message in a different subscription. The types that we use to send and receive messages are the same as the types used to send and receive messages using queues. However, the way we construct them is slightly different. When constructing a `ServiceBusSender` we pass the topic name to the `CreateSender` method. When constructing a `ServiceBusReceiver`, we have to pass both the topic name *and* the subscription name.
```

Comment:
Should we mention that subscriptions won't see messages in the topic that exist before the subscription was created - or point to service does?   That seems to be a common confusion in SO questions.